webpack.config.ts

```diff
@@ -448,6 +448,8 @@ const appConfig: Configuration = {
       crypto: require.resolve('crypto-browserify'),
       // `yarn why` says this is only needed in dev deps
       string_decoder: false,
+      // For framer motion v6, might be able to remove on v11
+      'process/browser': require.resolve('process/browser'),
```

Comment:
helps this version of framer-motion resolve the process/browser package that we are already using